src/include/execution/vm/bytecode_emitter.h

```diff
@@ -430,38 +421,34 @@ class BytecodeEmitter {
                              uint32_t index_oid, LocalVar col_oids, uint32_t num_oids);
 
   /**
-   * Emit code to free an index iterator
-   * @param bytecode bytecode used to free an index iterator
-   * @param iter index iterator to free
+   * Initialize a StringVal from a char array
+   * @param bytecode bytecode to emit
+   * @param out where to store the result
+   * @param length length of the string
+   * @param data pointer to the char array
    */
-  void EmitIndexIteratorFree(Bytecode bytecode, LocalVar iter);
+  void EmitInitString(Bytecode bytecode, LocalVar out, uint64_t length, uintptr_t data);
```

Comment:
This will have to be removed ...

Reply:
This will be removed in a future PR that @amlatyrngom is working on to implement parameters.